src/com/hms/flexyosisoftconnector/DataPoster.java

```diff
@@ -40,6 +40,12 @@ public void run() {
           case OSIsoftConfig.piwebapi:
             retval = OSIsoftServer.postBatch(payload);
             break;
+          case OSIsoftConfig.ocs:

```

Comment:
should this be `case OSIsoftConfig.OCS`? It is in ALL_CAPS_FORMAT in the OSIsoftConfig class

Reply:
the refactor feature missed it :(